src/studio/src/designer/frontend/ux-editor/containers/Container.tsx

```diff
@@ -761,36 +748,27 @@ export class ContainerComponent extends React.Component<
   };
 
   public renderContainer = (id: string, index: number): JSX.Element => {
-    const DroppableDraggableContainer =
-      require('./DroppableDraggableContainer').default;
     const canDrag = !this.state.activeList.find(
       (element: any) => element.id === id,
     );
     return (
       <DroppableDraggableContainer
         id={id}
         index={index}
-        baseContainer={false}
+        isBaseContainer={false}
         parentContainerId={this.props.id}
         canDrag={canDrag}
-        onDropComponent={this.props.onDropComponent}
-        onMoveComponent={this.props.onMoveComponent}
-        onDropContainer={this.props.onDropContainer}
-        onMoveContainer={this.props.onMoveContainer}
-        getIndex={this.getStatefulIndexOfContainer}
```

Comment:
SonarCloud also reports that `getStatefulIndexOfContainer` is not in use anymore and can be removed (ca line 216)